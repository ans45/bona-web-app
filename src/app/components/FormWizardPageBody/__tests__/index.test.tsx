import React from 'react';
import { render } from '@testing-library/react';
import { FormWizardPageBody } from '../';

describe('<FormWizardPageBody  />', () => {
  it('should match snapshot', () => {
    const { container, getByText } = render(
      <FormWizardPageBody>Hello World!</FormWizardPageBody>,
    );

    expect(getByText('Hello World!')).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
