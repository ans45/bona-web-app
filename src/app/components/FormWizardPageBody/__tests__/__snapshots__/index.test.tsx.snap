// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormWizardPageBody  /> should match snapshot 1`] = `
<div
  class="MuiBox-root MuiBox-root-1"
>
  <p
    class="MuiTypography-root MuiTypography-body1"
  >
    Hello World!
  </p>
</div>
`;
